server/main_loop.py

```diff
@@ -13,16 +13,18 @@
 if TICK_INTERVAL > 0:
     @scheduler_app.task(f"every {TICK_INTERVAL} seconds")  
     async def run_tick():
-        await tick()        
+        await tick()
 
 app = FastAPI.conserver_app
 app.scheduler = scheduler_app
 
+redis_mgr.create_pool()
```

Comment:
Why is the redis pool being created here?  This should not be necessary.  The redis pool should be created by the lifecycle startup event, which should always be called before any of the API/entry points can be invoked.  If there are cases where the startup event is not invokde first, then we have another problem that should be addressed rather than swept under the carpet.

Reply:
We had split out the API into it's own docker service so that we could get uvicorn auto-reloading in dev.  We are going to revert this since we realize it has bigger architectural implications and warrants more discussion.  We are going to close this PR and resubmit these changes in separate smaller PRs.